app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -579,10 +595,28 @@ class CommonTriageQuestionsController @Inject() (
                 propertyAddress = None,
                 disposalDetailsAnswers = None,
                 acquisitionDetailsAnswers = None,
-                reliefDetailsAnswers = None,
+                reliefDetailsAnswers =
+                  if (furtherReturn)
```

Comment:
let's doub;e check if we can just unset other reliefs as well - otherwise we'll have to do extra work to the preserve `lettingRelief` value for period of admin